doc/ref_model/chapters/chapter07.rst

```diff
@@ -394,52 +394,116 @@ ZTA principles applied to Cloud infrastructure components are the following:
-
-A transparent software supply chain offers benefits for vulnerabilities remediation, but also for licensing management and it provides assurance of the source and integrity of components. In order to use the SBOM efficiently and spread its adoption, information must be generated and shared in a standard format. This format must be machine-readable to allow automation. The NTIA white paper `“Survey of existing SBOM formats and standards” <https://www.ntia.doc.gov/files/ntia/publications/ntia_sbom_formats_and_standards_whitepaper_-_version_20191025.pdf>`__ identifies two key formats covering baseline SBOM information: Software Package Data eXchange (SPX) and Software Identification Tags (SWID tags).
+-  Adopt a DevSecOps approach and rely on testing automation throughout the software build, integration, delivery, deployment, and runtime operation to perform automatic security check, as described in section ”Infrastructure as a Code Security”
 
-`SPDX <https://spdx.dev/>`__ is an open-source machine-readable format developed under the umbrella of the Linux Foundation. The `SPDX specification 2.2 <https://spdx.dev/specifications/>`__ has been published as the standard ISO/IEC 5962:2021. It provides a language for communicating the data, licenses, copyrights, and security information associated with software components. With the SPDX specification 2.2, multiple file formats are available: YAML, JSON, RDF/XML, tag\:value flat text, and xlss spreadsheets.
+Open-Source Software Security
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-`SWID Tags <https://nvd.nist.gov/products/swid>`__ is an international XML-based standard used by commercial software publishers and has been published as the standard ISO/IEC 19770-2. The specification defines four types of SWID tags: primary, patch, corpus, and supplemental to describe a software component.
+Open-source code is present in Cloud Infrastructure software from BIOS, host Operating System to virtualisation layer 
+components, the most obvious being represented by Linux, KVM, QEMU, OpenStack, and Kubernetes. Workloads components 
+can also be composed of open source code. The proportion of open-source code to an application source code can vary. 
+It can be partial or total, visible or not. Open-source code can be upstream code coming directly from open-source 
+public repositories or code within a commercial application or network function. 
```

Comment:
Often an open-source software customised by an organisation (telecom operator in our case) is also understood as "open-source".  I am in two minds about it but I think we should specify whether we consider this as an open-source or a proprietary source based on open-source (I would vote for the latter...)

Reply:
On the second thought, if we consider "code within a commercial application or network function" as open source (because of its open source) then perhaps we should include also an open-source code within "a bespoke application or network function"?

Reply:
I vote also for  "proprietary source based on open-source" in this case.